src/main/scala/chiseltest/internal/BackendInterface.scala

```diff
@@ -11,13 +11,13 @@ import scala.collection.mutable
 trait AbstractTesterThread
 
 class TesterThreadList(protected val elts: Seq[AbstractTesterThread]) {
-  def toSeq(): Seq[AbstractTesterThread] = elts
+  def toSeq: Seq[AbstractTesterThread] = elts
 
-  def join() {
+  def join: Unit = {
     Context().backend.doJoin(elts, None)
   }
```

Comment:
I actually think `join` should keep the `()`. The idea is that if a 0-arity function has side-effects, it should have the `()`. `Iterator.hasNext` vs. `Iterator.next()` is a good example: https://www.scala-lang.org/api/2.13.8/scala/collection/Iterator.html

Reply:
I think you are right. I will change it.

Reply:
changed